api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -971,6 +979,33 @@ public Concept getReasonPatientNotFoundByActivist2ndVisitConcept() {
     return getConcept(uuid);
   }
 
+  // concept_id = 23723
+  public Concept getTbGenexpertTest() {
```

Comment:
This is already available in `TbMetadata`, we don't have to create it again here.